test/encryption_tests_v4.cpp

```diff
@@ -89,11 +89,41 @@ void crossPlatformConsistencyCheck_Base64UrlTest(const std::vector<std::uint8_t>
     }
 }
 
+void ValidateAdvertisingToken(const std::string& advertisingTokenString, IdentityScope identityScope, IdentityType identityType)
```

Comment:
Is there an example where this gets called with EUID and/or phone number?

Reply:
Sort of. There's these tests: https://github.com/IABTechLab/uid2-client-cpp11/blob/master/test/encryption_tests_v4.cpp#L363

This function takes those args to keep it in line with the original Operator tests:
https://github.com/IABTechLab/uid2-operator/blob/bdfcda4b7592a6c2fb65d676621c05de601954c1/src/test/java/com/uid2/operator/UIDOperatorVerticleTest.java#L757